lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1717,7 +1717,16 @@ static std::string GetSDKPath(std::string m_description, XcodeSDK sdk) {
   }
 
   std::string sdk_path = sdk_path_or_err->str();
-  LOG_PRINTF(GetLog(LLDBLog::Types), "Host SDK path: `%s` (XcodeSDK: %s)",
+  // GetSDKRoot reports no SDK as an empty string.
+  if (sdk_path.empty()) {
+    std::string sdk_spec = sdk.GetString().str();
+    Debugger::ReportError("Could not find an SDK for \"" + sdk_spec + "\".");
+    HEALTH_LOG_PRINTF("Could not find an SDK for \"%s\". Try to verify that "
```

Comment:
Also, Apple style uses common contractions, such as using _couldn't_ in place of _could not_.

```suggestion
    HEALTH_LOG_PRINTF("LLDB couldn't find an SDK for \"%s\". Try to verify that "
```